3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehousespec/rest/v1/dto/product/ProductCartResponse.java

```diff
@@ -0,0 +1,6 @@
+package com._6core.platform.warehousespec.rest.v1.dto.product;
+
+import java.math.BigInteger;
+
+public record ProductCartResponse(
```

Comment:
Cart? There should no cart-related info be in this service.
```suggestion
public record ProductCartResponse(
```

Reply:
@sokimaaa, I thought about this dto as a cart of the product. (https://mypresta.eu/content/uploads/2017/04/add-to-cart-quantity-product-list-prestashop17.png)

Reply:
Card is a flat stiff usually small and rectangular piece of material (such as paper, cardboard, or plastic) usually bearing information. e.g. Credit Card.